app/clients/file_client.rb

```diff
@@ -13,7 +13,7 @@ class FileClient
   # - the contents of the file.
 
   def self.save(user_id, folder, filename, content)
-    filepath = self.filepath user_id, folder, filename
+    filepath = filepath user_id, folder, filename
```

Comment:
@amatriain that wasn't the reason behind this change, it wasn't because of some style guide rule, it was because it breaks some tests with:

> NoMethodError:
>       private method `filepath' called for FileClient:Class

It seems that if you use self here, instead of just calling the method as you are already "in self", Ruby thinks you are calling it from outside the class. I'm still trying to understand exactly why, it's the same thing that is happening on #11